include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_merge_sort.h

```diff
@@ -16,16 +16,17 @@
 #ifndef _ONEDPL_PARALLEL_BACKEND_SYCL_MERGE_SORT_H
 #define _ONEDPL_PARALLEL_BACKEND_SYCL_MERGE_SORT_H
 
-#include <cmath>     // std::log2
-#include <limits>    // std::numeric_limits
-#include <cassert>   // assert
-#include <utility>   // std::swap
-#include <cstdint>   // std::uint32_t, ...
-#include <algorithm> // std::min
+#include <cmath>       // std::log2
```

Comment:
I think we can remove this include now that we use `sycl::ctz`

Reply:
Done